packages/ui/app.js

```diff
@@ -0,0 +1,18 @@
+var express = require("express");
```

Comment:
```suggestion
const express = require("express");
```

Reply:
Sure will do every where possible in this file 👍